view.py

```diff
@@ -12,20 +10,14 @@ def boot_message(self,role,num_applicants):
     def view_role(self,role):
         print(f"Role title: {role.job_title}")
         print(f"Role ID: {role.job_id}")
+        
+    def view_applicant_details(self,applicant):
+            print(f"Details for {applicant.name}: ")
+            print(f"CV Path: {applicant.cv}")
+            print(f"Scores: {applicant.scores}")
 
-    def view_applicants(self,applicants):
-        print(f"Here are all of the applicants:")
-        print()
-
-        for applicant in applicants:
-          i = applicants.index(applicant)
-          print(f"{i+1}.Name: {applicant.name}")
-          print(f"CV Path: {applicant.cv}")
-          print(f"Scores: {applicant.scores}")
-          print()
-    
-    def view_shortlist(self,shortlist):
-        print(f"Shortlist for {shortlist.role.job_title}:")
+    def view_applicants_list(self,shortlist):
+        print(f"List of applicants for {shortlist.role.job_title}:")
 
         for applicant in shortlist.applicants:
             i = shortlist.applicants.index(applicant)
```

Comment:
Here, you're searching the list each iteration of the loop for find its index. Python offers a good way to get the index of items when iterating. Look at the `enumerate` function.